### PR TITLE
feat: added files for Slider component

### PR DESCRIPTION
**Pull request by Ranjeetk489**

### CODE REVIEW

File: components/Slider/slider.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  border-radius: 5px;
+  padding: 20px;
+  box-shadow: 0 15px 30px rgba(0, 0, 0, 0.3);
+}
+.image-slider {
+  position: relative;
+}
+img {
+  width: 460px;
+  height: 280px;
```
Comment (sanu29) [inline components/Slider/slider.css:52]: Use rem

**Review by sanu29 — APPROVED** [review]
Good to merge

File: components/Slider/slider.css (hunk 1 above)
Comment (sanjitsarkar) [reply components/Slider/slider.css:52]: Use rem

**Review by sanjitsarkar — APPROVED** [review]
Use rem in place of px

**Pull request merged**